prov/efa/src/efa_env.c

```diff
@@ -41,29 +41,6 @@ struct efa_env efa_env = {
 	.internal_rx_refill_threshold = 8,
 };
 
-/**
- * @brief Define FI_EFA_USE_DEVICE_RDMA as a configuration parameter
- *
- * This function fetches the default value of using EFA device's
- * RDMA capability and defines it as a configuration parameter.
- */
-void efa_env_define_use_device_rdma()
-{
-	char *str = "";
-
-	/* Specify the help info about the usage of RDMA in the device. */
-	if (!efa_device_support_rdma_read()) {
```

Comment:
it seems like you can still run this check, just not as part of `efa_env_define()`.

Reply:
Yes. We have a check during endpoint open https://github.com/ofiwg/libfabric/blob/5dd902300d2c5cdf27f436ffe430d4217c093f5c/prov/efa/src/rdm/efa_rdm_util.c#L63-L70